### PR TITLE
Fix delated volume

### LINKED ISSUE #1097: Delay on volume changes
When using the softwaremixer, there is a delay when changing the volume or muting until the change actually occurs.

This was introduced by 0cfd69432e8ba9781e1ff1aa40c22ef7d922c4e1.

### PR DESCRIPTION
**Pull request by adamcik**
Fixes #1097

### CODE REVIEW

**Comment by jodal** [comment]
I assume you intended this for the v1.1 milestone.

File: docs/changelog.rst [hunk 1]
```diff
@@ -14,6 +14,16 @@ Core API
 - Calling :meth:`mopidy.core.library.LibraryController.search`` with ``kwargs``
   as the query is no longer supported (PR: :issue:`1090`)
 
+Audio
+-----
+
+- Software volume control has been reworked to greatly reduce the delay between
+  changing the volume and the change taking effect. (Fixes: :issue:`1097`)
+
+- Software volume is no longer tied to the Pulse audio volume when using
```
Comment (jodal) [inline docs/changelog.rst]: s/Pulse audio volume/PulseAudio application volume/

Note: both change in capitalization and adding of "application"

File: mopidy/audio/actor.py [hunk 2]
```diff
@@ -229,24 +194,16 @@ def get_volume(self):
...
-            gst_logger.debug('Notify volume: %s', self._last_volume / 100.0)
-            self._mixer.trigger_volume_changed(self._last_volume)
-
-    def _mute_changed(self, element, property_):
-        old_mute, self._last_mute = self._last_mute, self.get_mute()
-        if old_mute != self._last_mute:
-            gst_logger.debug('Notify mute: %s', self._last_mute)
-            self._mixer.trigger_mute_changed(self._last_mute)
+        result = self._element.set_property('mute', bool(mute))
+        if result:
```
Comment (jodal) [inline mopidy/audio/actor.py]: Why check result here and not for set_volume?

Comment (adamcik) [inline mopidy/audio/actor.py]: Because that is what the old code does, I don't know why we check for mute, but not for volume.

**Comment by adamcik** [comment]
I was intending we get this into the next 1.0.x actually. Was unsure on if I should have done this against a 1.0.x branch and done the changelog against 1.0.1 and then merged into develop or the other way around.

**Comment by jodal** [comment]
Looks good to me.

We already have a `v1.0.x` branch. Can you make new PR against that branch, with a changelog for v1.0.1?

I find it easier to be sure everything is in the right place by having:

- minor docs fixes in master
- bugfixes in v1.0.x
- features in develop

And then regularly merge master -> v1.0.x -> develop.

**Comment by adamcik** [comment]
Sure, I'll rebase against v1.0.x, edit the changelog and open the correct PR for this.

**Comment by adamcik** [comment]
Creating a new PR based of correct branch.

**Pull request closed without merging**